backend/functions/cases/cases.integration.test.ts

```diff
@@ -4,13 +4,16 @@ dotenv.config();
 
 describe('Integration Test for the cases Azure Function to call get cases', () => {
   let functionUrl;
+  let slotName;
   beforeAll(() => {
     functionUrl = process.env.CASES_FUNCTION_URL;
+    slotName = process.env.SLOT_NAME ?? 'self';
   });
 
   test('cases azure function should return success when retrieving cases', async () => {
     let casesResponse;
-    await fetch(`${functionUrl}/cases`, {
+    const targetURL = `${functionUrl}/cases?x-ms-routing-name=${slotName}`;
+    await fetch(targetURL, {
```

Comment:
This gets flagged by Veracode, a possible refactor is to default to self and check a boolean to determine if the slot deployment environment is not production. If not explicitly pass in the slotName as staging.

Reply:
Tried several refactoring but the problem seems to be related to the functionUrl that needs to be passed in. @amorrow-flexion mention that this was an existing known Veracode finding that was ignored. Baseline needs to be updated.